core/src/test/scala/sbt/librarymanagement/ResolverExtraTest.scala

```diff
@@ -0,0 +1,43 @@
+package sbt.librarymanagement
+
+import verify.BasicTestSuite
+import scala.annotation.nowarn
+
+@nowarn // Necessary because our test cases look like interpolated strings.
+object ResolverExtraTest extends BasicTestSuite {
```

Comment:
One weirdness that I haven't had the courage to look into: this was initially a `UnitSpec`, like `ResolverTest`, but tests didn't get executed.

Not sure whether this is a bug or working as intended, so I didn't change it. Note that `ResolverTest` isn't getting executed, at least on my local setup.

Reply:
Not sure what it could be. Maybe it's due the fact that some are ScalaTest that uses class and others are verify that uses object?

Reply:
Honestly not a clue. There's a `verify` runner explicitly registered in the build, which I initially thought might override existing ones, but I see that *some* ScalaTest tests are being executed so... very confused.